docker/deployment/app-deployment.yaml

```diff
@@ -178,3 +178,37 @@ spec:
               secretKeyRef:
                 key: LD_TRUST_TOKEN
                 name: ld-trust-token
+      - name: lfmerge
+        # TODO: Decide on a verion-numbering policy for LfMerge releases
+        image: ghcr.io/sillsdev/lfmerge:latest
+        imagePullPolicy: Always
+        volumeMounts:
```

Comment:
if there are strict volume permissions, we'll need to make sure we have an init container to get those correct.

Reply:
Agreed. The init container we currently have should work fine, but I'll work out the issues in the staging environment if it doesn't.